sdk/eventgrid/azure-eventgrid/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 2.0.0b1 (Unreleased)
+
+  - Placeholder - NEEDS TO BE CHANGED
```

Comment:
Were you planning to swap this out with some representative snippet?  or do it as an intentional step later once we're more settled.

Reply:
I'll do this in a later pr once things are settled - this one is sort of supposed to be a work in progress pr.

Reply:
SGTM.  Let's make sure we have an issue tracking to put docs in before september release.  (Blame this on my distrust of my own memory to ensure this doesn't fall off the bus, obvious though it may be.)

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/13223